openupgrade_scripts/scripts/website/14.0.1.0/post-migration.py

```diff
@@ -26,8 +28,18 @@ def website_cookie_notice_post_migration(env):
             env["ir.ui.view"].browse(view_ids).unlink()
 
 
+def convert_html_string(env):
+    website_views = env["ir.ui.view"].search(
+        [("type", "=", "qweb"), ("website_id", "!=", False)]
+    )
+    for view in website_views:
+        view.arch_db = convert_html_string_13to14(view.arch_db)
+        view.write({"arch_db": view.arch_db})
```

Comment:
You're already doing it via assignation in the previous line
```suggestion
```

Reply:
thanks for the clarification :)